src/messages.js

```diff
@@ -421,4 +421,23 @@ export const sayDiamondAlready = (isModerator, wasModerator) => {
 
     const [, message] = messageMap.find(([condition]) => condition) || [];
     return message || `diamonds are forever!`;
+};
+
+/**
+ * @summary builds a "number of positions" message
+ * @param {BotConfig} _config bot configuration
+ * @param {Election} election current election
+ * @param {string} _text message content
+ * @returns {string}
+ */
+export const sayNumberOfPositions = (_config, election, _text) => {
+    const { numPositions = 0 } = election;
+
+    const suffix = pluralize(numPositions, "s", "");
+    const pastBe = pluralize(numPositions, "were", "was");
```

Comment:
We could likely use `Intl` here, but it's a bit too convoluted for such a simple task